packages/sources/src/components/form/models/form.model.ts

```diff
@@ -0,0 +1,7 @@
+import { FormGroup } from './form-group.model';
+
+export interface Form<TemplateFnReturnType> {
+  mode?: 'vertical' | 'horizontal';
+
+  formGroups: FormGroup<TemplateFnReturnType>[];
```

Comment:
In aanvulling op vorige comment:

```suggestion
  formGroups: FormGroup<TemplateFnReturnType>[] | TemplateFnReturnType;
```

Reply:
aangepast